config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json

```diff
@@ -39,10 +39,10 @@
     "item.resourcefulbees.lead_honeycomb": "Lead Honeycomb",
     "item.resourcefulbees.lead_bee_spawn_egg": "Lead Bee Spawn Egg",
     "entity.resourcefulbees.lead_bee": "Lead Bee",
-    "block.resourcefulbees.netherite_honeycomb_block": "Ancient  Debee Honeycomb Block",
-    "item.resourcefulbees.netherite_honeycomb": "Ancient  Debee Honeycomb",
-    "item.resourcefulbees.netherite_bee_spawn_egg": "Ancient  Debee Spawn Egg",
-    "entity.resourcefulbees.netherite_bee": "Ancient  Debee",
+    "block.resourcefulbees.netherite_honeycomb_block": "Netherite Bee Honeycomb Block",
+    "item.resourcefulbees.netherite_honeycomb": "Netherite Bee Honeycomb",
```

Comment:
Late to the party here, but shouldn't these be changed to "Netherite Honeycomb" and "Netherite Honeycomb block" to be consistent with the other mineral bees?

Reply:
Fixed in #1766 ^^

Thanks!